docs: document how to skip authentication (#65)
kenji-miyake/setup-git-cliff: README.md

## Action inputs and outputs

Refer to [action.yaml](./action.yaml).

> ### Skip Authentication
> 
> By default, this action will use the `${{ github.token }}` context variable to authenticate to GitHub.
> 
> This could result in a `Bad credentials` exception if the token used by the job is not valid for
> <https://github.com/>. An example where this happens is if this action is running on self-hosted
> runner that's connected to a GitHub Enterprise Server instance.
> 
> In case, you can explicitly set the token to `null` to perform an unauthenticated request:
> 
> ```yaml
> - name: Set up git-cliff
>   uses: kenji-miyake/setup-git-cliff@v1.1.0
>   with:
>     token: null
> ```

Reviewer: @pmeyerra How about this?

- Use sentence case.
- To be shorter.
  - Reference: https://developers.google.com/style/translation#write-short,-clear,-and-precise-sentences
- Mention the goal first.
  - Reference: https://developers.google.com/style/clause-order
- Use v1.

``````suggestion
### Skip authentication

By default, this action will use the `${{ github.token }}` context variable to authenticate to GitHub.

This could result in a `Bad credentials` exception if the action is running on a self-hosted runner connected to a GitHub Enterprise Server instance.

To avoid this, you can explicitly set the token to `null` to perform an unauthenticated request:

```yaml
- name: Set up git-cliff
  uses: kenji-miyake/setup-git-cliff@v1
  with:
    token: null
```
``````

Author: That looks good, thanks for the references and for improvements!
